pkg/dao/repository_configs.go

```diff
@@ -518,6 +518,17 @@ func ModelToApiFields(repoConfig models.RepositoryConfiguration, apiRepo *api.Re
 	apiRepo.RepositoryUUID = repoConfig.RepositoryUUID
 	apiRepo.Snapshot = repoConfig.Snapshot
 
+	apiRepo.LastSnapshotUUID = repoConfig.LastSnapshotUUID
+
+	if repoConfig.LastSnapshotUUID != "" {
```

Comment:
```suggestion
	if repoConfig.LastSnapshot != nil {
```

would be safer, if for some reason LastSnapshotUUID was non-blank, but LastSnapshot was nil (such as someon forgot to preload it), it would panic trying to dereference the nil pointer